part-3/app/controllers/bids.rb

```diff
@@ -0,0 +1,12 @@
+get '/bids/active' do
+  @items = Item.where("start_time < ? AND end_time > ?", Time.now, Time.now)
```

Comment:
this could be rewritten using [AR scopes](http://guides.rubyonrails.org/active_record_querying.html#scopes) like this: 

```ruby
class Item < ActiveRecord::Base
  scope :active, ->{ where("start_time < ? AND end_time > ?", Time.now, Time.now) }
end
```

and then here you could just say

```ruby
@items = Item.active
```